app/models/outgoing_message.rb

```diff
@@ -344,8 +346,10 @@ def get_text_for_indexing(strip_salutation = true, opts = {})
       text = body(opts).strip
     end
 
-    # Remove salutation
-    text.sub!(/Dear .+,/, "") if strip_salutation
+    if strip_salutation && public_body
+      salutation = self.class.default_salutation(public_body)
+      text.sub!(/#{salutation}\s*/, '')
```

Comment:
Depending on the translation we might need

```suggestion
      text.sub!(/#{Regexp.escape(salutation)}\s*/, '')
```

Reply:
Fixed in 4948073b3